testpar/t_filter_read.c

```diff
@@ -194,9 +205,8 @@ test_filter_read(void)
     unsigned      disable_partial_chunk_filters; /* Whether filters are disabled on partial chunks */
     herr_t        hrc;
     const char   *filename;
-#ifdef H5_HAVE_FILTER_FLETCHER32
```

Comment:
H5_HAVE_FILTER_FLETCHER32 is never defined anywhere, so I re-enabled this part of the test